sdk/compiler/damlc/daml-package-config/src/DA/Daml/Package/Config.hs

```diff
@@ -129,27 +131,27 @@ overrideSdkVersion pkgConfig = do
         Just (Left sdkVersionError) -> do
             hPutStrLn stderr $ unwords
                 [ "Warning: Using SDK version "
-                , V.toString (unwrapUnresolvedReleaseVersion (pSdkVersion pkgConfig))
+                , maybe "<omitted>" (V.toString . unwrapUnresolvedReleaseVersion) (pSdkVersion pkgConfig)
```

Comment:
What prevents us from importing `unresolvedBuiltinSdkVersion` to give something like `<unresolvedBuiltinSdkVersion here> (inferred from DPM)` in this case, rather than `<omitted>`?